reana_commons/yadage.py

```diff
@@ -55,7 +55,8 @@ def yadage_load_from_workspace(workspace_path, reana_specification, toplevel, **
 
     :returns: Updated reana specification with loaded `yadage` workflow.
     """
-    workflow_workspace = "{0}/{1}".format(SHARED_VOLUME_PATH, workspace_path)
+    # workflow_workspace = "{0}/{1}".format(SHARED_VOLUME_PATH, workspace_path)
+    workflow_workspace = workspace_path
```

Comment:
Similarly we could get rid of `workflow_workspace` and use `workspace_path` instead